src/find/matchers/mod.rs

```diff
@@ -892,6 +895,26 @@ fn build_matcher_tree(
                 config.version_requested = true;
                 None
             }
+            "-files0-from" => {
+                // The logic for the below if is to handle edge cases like `-files0-from -ok` etc.
+                if i >= args.len() - 1 {
+                    return Err(From::from(format!("missing argument to {}", args[i])));
+                }
+                parse_files0_args(config, args[i + 1])?;
+                i += 1;
+                None
+            }
+            // -ok and -okdir are yet to be implemented fully
```

Comment:
Sorry, I didn't realize these weren't even implemented originally.  You might as well just keep them not implemented, so every use of `-ok` will error.  Then it will be up to whoever implements `-ok` to make the interaction with `-files0-from` work ;)